btcd.go

```diff
@@ -266,9 +266,9 @@ func btcdMain(serverChan chan<- *server) error {
 	server, err := newServer(cfg.Listeners, cfg.AgentBlacklist,
 		cfg.AgentWhitelist, db, activeNetParams.Params, interrupt)
 	if err != nil {
-		// TODO: this logging could do with some beautifying.
-		btcdLog.Errorf("Unable to start server on %v: %v",
-			cfg.Listeners, err)
+		// Format error message more clearly with better details
+		btcdLog.Errorf("Failed to initialize server: %v", err)
+		btcdLog.Errorf("Unable to start server with configured listeners: %v", cfg.Listeners)
```

Comment:
Could you swap the order of these two log lines.  I think the output will make more logical sense that way.